lib/lsperfm/defaults/config/simple_grok.conf

```diff
@@ -0,0 +1,13 @@
+input {
+	stdin { type => "apache" }
```

Comment:
just a matter of consistency, maybe format all .conf files in the same way (e.g. 2 spaces)?